lib/public/components/Filters/common/filters/checkboxFilter.js

```diff
@@ -40,13 +41,13 @@ export const checkboxFilter = (name, values, isChecked, onChange, additionalProp
     ])));
 
 /**
- * A general component for generating checkboxes.
+ * Component comprasing list of checkboxes defined by list of options provided by given selection model
```

Comment:
Display a filter composed of checkbox listing pre-defined options

I don't understand the `comprasing`.

Anyway, this seems quite generic, not limited to filter.
However, don't you use selectionDropdown?

Reply:
but for inline list of checkboxed, not dropdown, applied your docs, there was typo, should be `comprising`